test/fail_compilation/diag13609b.d

```diff
@@ -1,8 +1,9 @@
 /*
 TEST_OUTPUT:
 ---
-fail_compilation/diag13609b.d(9): Error: { } expected following struct declaration
-fail_compilation/diag13609b.d(9): Error: declaration expected, not ':'
+fail_compilation/diag13609b.d(11): Error: basic type expected, not EOF
```

Comment:
How is that not a regression?  It seems like it should be very easy to not attempt to parse base classes when it's not a class/interface.  That extra error message is not helpful, and doesn't make any sense.